pylib/spinnaker/codelab_config.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import spinnaker.yaml_util as yaml_util
+
+def configure_codelab_igor_jenkins():
+    """Configures Igor to be enabled and to point to the codelab jenkins instance.
+
+    """
+    yaml_binds = yaml_util.YamlBindings()
+    update_dict = {
+        'jenkins': {
+            'defaultMaster': {
+                'name': 'CodelabJenkins',
+                'baseUrl': 'http://localhost:9090',
+                'username': 'admin',
+                'password': 'admin'
+            }
+        },
+        'igor': {
+            'enabled': 'true'
+        }
+    }
+    local_yml = '/opt/spinnaker/config/spinnaker-local.yml'
+    yaml_binds.update_yml_source(local_yml, update_dict)
```

Comment:
And you can even inline the update dict and local yml, pass by name since that's all this function is.